backend/app/models/group.rb

```diff
@@ -0,0 +1,82 @@
+# groups table
+class Group < ActiveRecord::Base
+
+  validates :name,        presence: true
+  validates :description, presence: true
+
+  # Return all groups.
+  #
+  # @return all groups
+  def self.find_all
+    Group.order(:name)
+  end
+
+  # Return all groups beginning with fitst letter l ('*' as non-alphanumeric wildcard).
+  #
+  # @return all groups beginning with fitst letter l ('*' as non-alphanumeric wildcard)
+  def self.find_letter(l)
```

Comment:
this name suggests you are searching for the letter rather than for the groups with names starting with the letter.

Reply:
Done.